lib/index.js

```diff
@@ -17,11 +17,8 @@ function processArrayElement(value) {
   return segment;
 }
 
-function buildSrc(config) {
-  if (!config) {
-    config = {};
-  }
-
+function buildSrc(_config) {
+  const config = Object.assign({}, _config);
```

Comment:
Just shallow clone since `delete config.{src,min}` below.  Don't want to directly mutate config object passed into buildSrc.